.codecov.yml

```diff
@@ -203,14 +215,6 @@ coverage:
         target: 75
         flags:
         - pulsar
-      PureFA:
```

Comment:
Can you please rebase with the `master` branch so that these additions get pulled in?